### PR TITLE
Logo and bugs

### PR DESCRIPTION
**Pull request by orhovy**

### CODE REVIEW

File: src/main/kotlin/com/bridgecrew/CheckovResult.kt [hunk 1]
```diff
@@ -58,8 +60,26 @@ fun groupResultsByResource(results: ArrayList<CheckovResult>, project: Project,
...
+    try {
+        val failedRaw = resultsObj.get("failed")
+        if (failedRaw == 0) {
+            throw CheckovResultException("Results empty")
+        }
+        } catch (e : JSONException) { }
+    try {
+        val summary = resultsObj.getJSONObject("summary")
+            val failedSummary = summary.get("failed")
+            val parsingErrorSummary: Int =summary.getInt("parsing_errors")
```
Comment (orhovy) [inline src/main/kotlin/com/bridgecrew/CheckovResult.kt:72]:
```suggestion
            val parsingErrorSummary: Int = summary.getInt("parsing_errors")
```

File: src/main/kotlin/com/bridgecrew/CheckovResult.kt [hunk 2]
```diff
@@ -58,8 +60,26 @@ fun groupResultsByResource(results: ArrayList<CheckovResult>, project: Project,
...
+        val summary = resultsObj.getJSONObject("summary")
+            val failedSummary = summary.get("failed")
+            val parsingErrorSummary: Int = summary.getInt("parsing_errors")
+            if (parsingErrorSummary > 0 ) {
+                throw CheckovResultParsingException("Checkov parsing error")
+            }
+            if (failedSummary == 0){
+                throw CheckovResultException("Results empty")
+            }
+    } catch (e : JSONException) { }
```
Comment (yuvalyacoby) [inline src/main/kotlin/com/bridgecrew/CheckovResult.kt:79]: not sure I understand this. why throw exception and then catch and do nothing ?

**Review by yuvalyacoby — APPROVED** [review]
looks good

File: src/main/kotlin/com/bridgecrew/CheckovResult.kt (hunk 2 above)
Comment (orhovy) [reply src/main/kotlin/com/bridgecrew/CheckovResult.kt]: Removed it , it was unnecessery

**Pull request merged**